### DISCUSSION TITLE
About q-function

### DISCUSSION THREAD

**Discussion by kuaiyiLi** [autonlab/auton-survival#97] [Q&A]
Is there a derivation of q function?Can I share it? Thank you!

**Comment by chiragnagpal** [comment]
Are you referring to the Q function for the Deep Cox Mixtures paper?

**Reply by kuaiyiLi** [reply]
Yes!

**Reply by chiragnagpal** [reply]
The Q function for any general latent variable model is given as follows:

Q(theta) = E(Z|{All other variables}) [ log_likelihood({all other variables}) ]

**Comment by ghost** [comment]
## ?? Reddit ????? - OpenClaw MCP Ecosystem 175+ votes

**??:** r/mcp

> OpenClaw ???? 9 ??? MCP ???,??? Cloudflare Workers,?? Free tier + Pro API keys??????????? OpenClaw MCP ??!

---
*?? Reddit*

**Comment by ghost** [comment]
Reddit: OpenClaw MCP Ecosystem - 175+ votes from r/mcp. 9 remote MCP servers on Cloudflare Workers!
